portfolio/src/main/webapp/index.html

```diff
@@ -238,6 +241,32 @@ <h2>Connect with me</h2>
   </section>
 </div>
 
+<!-- Modal for setting nickname -->
+<div class="modal fade" id="nickname-modal" tabindex="-1" role="dialog"
+    aria-labelledby="nickname-modal-title" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="nickname-modal-title">Set Nickname</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <form action="/login" method="post" id="nickname-form">
+        <div class="modal-body">
+            <label for="nickname">Nickname</label>
+            <input class="form-control" id="nickname" aria-describedby="nickname-info"
+                placeholder="Enter Nickname" name="nickname" required>
+            <small id="nickname-info" class="form-text text-muted">This nickname will be associated with your account.</small>
```

Comment:
Avoid using HTML elements for specific styling (e.g. `<i>`, `<small>`, `<center>`, etc). Use CSS instead.

See https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Reply:
Thanks for the reference. Changes made.